### ISSUE TITLE
Bot times out if it receives a message it doesn't know how to handle

### ISSUE THREAD

**Issue by 8549** [telegraf/telegraf#1089]
## Context

I am deploying my bot on Google Cloud Functions. I manually set up the webhook url after deployment. Privacy mode is enabled and bot is not admin in the groups i have added it in. The bot works successfully 3/4 times right after deployment then starts to time out and stops responding, with a lot of accumulating `pending_update_count` returned from `getWebhookInfo`. This is because if it receives a command that i haven't registered to listen for, or someone replies to the bot's messages, instead of ignoring the update (i mean just sending a status 200 to Telegram servers so the update is marked as processed), it idles waiting for something.

**tl;dr:** bot is missing a safety-net, catch-all mechanism to ignore and report as processed updates it doesn't understand (e.g. replies to its messages or commands that don't exist)

* Telegraf.js Version: 3.38.0
* Node.js Version: 12.18.2
* Operating System: Ubuntu 20.04

## Expected Behavior

Bot should process updates even if it doesn't know what to do with the content of the update, just by responding with a 200 status code or some other kind of acknowledgment to Telegram's request

## Current Behavior

Updates with messages that the bot can't understand stay in the queue and don't get processed, causing the bot to time out and never process the queue

### Steps to Reproduce

Minimum working example:

```javascript
'use strict';

const { Telegraf } = require('telegraf');

const bot = new Telegraf(process.env.TELEGRAM_TOKEN, { telegram: { webhookReply: true } });

bot.telegram.getMe().then((botInfo) => {
  bot.options.username = botInfo.username;
});


bot.start((ctx) => ctx.reply(`Bot started`));
bot.catch((err, ctx) => {
  console.error(err);
  return ctx.reply('An error occurred.');
});

exports.start = async (req, res) => {
  return bot.handleUpdate(req.body, res);
}
```

Deployed with `gcloud functions deploy myBot --region europe-west1 --trigger-http --timeout 10 --entry-point start --env-vars-file env.yaml --allow-unauthenticated --runtime nodejs12`

### Failure Logs

Google Cloud Logging shows this:
![image](https://user-images.githubusercontent.com/15671077/87854148-0ba14700-c910-11ea-935f-b619e028dea6.png)

### Workaround

I found that by adding this line after bot.start(...) the problem goes away:
`bot.on('message', (ctx, next) => ctx.reply('Ignore this', Extra.inReplyTo(ctx.message.message_id)));`
But this is more a workaround than a fix, as bot should gracefully ignore the updates it hasn't been programmed for.
Additionally, sometimes the workaround `.on('message')` middleware is hit even if the bot receives a correct command, so it is not even a perfect workaround.

I also tried with `bot.on('message', (ctx, next) => next());` but that didn't work.

**Comment by kidonng** [comment]
I come up with a dumb idea, it works, if you don't mind possible error message in your serverless function logs.

```js
bot.on('message', async (ctx) => {
    const { url } = await ctx.telegram.getWebhookInfo()
    await ctx.telegram.deleteWebhook()
    await ctx.telegram.getUpdates()
    return ctx.telegram.setWebhook(url)
})
```

**Comment by 8549** [comment]
> Additionally, sometimes the workaround `.on('message')` middleware is hit even if the bot receives a correct command, so it is not even a perfect workaround.

**Comment by kidonng** [comment]
> Additionally, sometimes the workaround `.on('message')` middleware is hit even if the bot receives a correct command, so it is not even a perfect workaround.

I have not encountered this issue, and I would assume your code is to blame (no offense). Even if I do, the issue still has nothing to do with my workaround. If it didn't help you, you should explain the exact reason.

**Comment by 8549** [comment]
Turns out i was deploying the wrong branch without your workaround; it works, thanks! I'll leave this open in case the devs have a more streamlined way to do this or might want to take inspiration and add a corresponding method to the framework.
